cli/cmd/install.go

```diff
@@ -419,6 +422,9 @@ func (options *installOptions) recordableFlagSet() *pflag.FlagSet {
 		&options.highAvailability, "ha", options.highAvailability,
 		"Enable HA deployment config for the control plane (default false)",
 	)
+	flags.StringVar(&options.nodeSelectorString, "node-selector", options.nodeSelectorString,
```

Comment:
I would rather not expose this functionality through `install`.

Reply:
Sure, removed!